ipld/src/selector/mod.rs

```diff
@@ -3,7 +3,7 @@
 
 mod empty_map;
 mod walk;
-pub use walk::{LinkResolver, Progress, VisitReason};
+pub use self::walk::*;
```

Comment:
```suggestion
pub use walk::*;
```

Reply:
why? This is more clear about what is being exported

Reply:
Ah, like, to indicate it's not from another crate or something? I thought you put it there by accident, didn't think about it too hard